test/test_namedtuple_return_api.py

```diff
@@ -0,0 +1,38 @@
+import os
+import re
+import yaml
+import unittest
+
+
+path = os.path.dirname(os.path.realpath(__file__))
+aten_native_yaml = os.path.join(path, '../aten/src/ATen/native/native_functions.yaml')
+whitelist = [
```

Comment:
Don't we intend to give all functions named outputs in short order? In this case, I don't feel like this test case is giving us much utility, except perhaps reminding us to carefully audit choice of named outputs. What do you think?

Reply:
@ezyang Yes, I added this test simply to remind people be careful on these names, because this might be a point people could easily forget. I'm OK with removing it if this won't be a problem.